README.md

```diff
@@ -1,27 +1,179 @@
+
+We have also ensured to test each part of the pipeline to ensure that they work and these can be viewed below
+
+- `test_extract.py` -> | Tests dataframe creation | Tests the URL with the cases | Tests the index to infinity function |
+- `test_transform.py` -> | Tests date returns correct bool | Tests date formats correctly w/ Parametrisation | Tests if judge titles are stripped | Tests if case numbers have been standardised w/ Parametrisation
+- `test_load.py` -> xxxxxx
+
+## Data Cleaning
+
+After fetching the data, the script performs cleaning operations to ensure consistency and data integrity. It standardises judges names and fixes the multiple different date\time formats when extracted from the pdfs to keep consistency within our database.
+
+## Database Interaction
+
+The script establishes a database connection using the provided environment variables and inserts the cleaned data into the Database. It constructs SQL query strings dynamically based on the cleaned data and executes them to insert the data into the database.
+
+## Logging and Error Handling
+
+The script utilises the logging module to log important events and errors during the execution process. This helps in debugging and monitoring the pipeline's performance.
+
+## BASH Database Scripts
+
+We have also implemented bash scripts to simplify and automate connection and manipulation of the database. These scripts depend on environmental variables. Ensure you have a .env file with the correct values for each variable. To run these scripts in the terminal, execute `bash [.sh script]`.
+
+### Connection
+
+For this, we have a `connect_db.sh` which connects directly to the PSQL database without needing a password.
+
+### Table Creation
+
+For this, we have a `insert_schema.sh` that runs the `schema.sql` into the database itself and adds all the tables with their corresponding keys.
+
+### Seeding Data
+
+For this, we have a `seed.sh` which runs the `seeds.sql` script into the database, inserting all the static data into the tables within the database as well.
+
+## Deployment
+
+Access to Dashboard via ECS: xxxxxx
+
+### Features
+- **XXXXXX:**
+- **XXXXXX:**
+- **XXXXXX:** 
+
+### Usage
+- XXXXXXXXX
+- XXXXXXXXX
+- XXXXXXXXX
+
+### Data Sources
+- **Database:** The app retrieves real-time judge and court data from a database using SQL queries.
+
+## Terraform
+
+This repository contains Terraform scripts to provision Lambda functions on AWS for various purposes related to managing plant data.
+
+- Lambda Functions for accessing AWS services `main.tf`
+  - This file will spin up the AWS services using the access and secret key.
+- Lambda Functions for running the pipeline `pipeline.tf`
+  - This file will create a lambda function which will run the whole pipeline file itself.
+- Lambda Functions for RDS Insertion `eventbridge.tf`
+  - This file will create a lambda function that spins up the event bridges that connect itself to the lambda function.
+ 
+### Dependencies
```

Comment:
Great use of a table to highlight what is needed by the user to run the terraform.